sections/plate_notation.tex

```diff
@@ -0,0 +1,39 @@
+\section{Plate Notation}
+As with \citet{author_topic_2012}, there are no document-topic distributions $\theta$ in our metadata models.
+Instead, each author and category has their own topic distribution.
+This is based on the assumption that authors prefer to write about specific topics, and that categories of the articles were chosen based on the content of the finished article or that local newspapers have their own unique topic preferences.
+
+Our topic models are \glspl{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have influence on both word distributions and other metadata.
+
+Our standard \gls{lda} model uses a similar notation to \citet{blei2003latent}, where a document $d$ is a vector of $N$ ID's, where the ID represents the given word in the original document. $Z_{d,n}$ is the topic assigned to a word $n$ within $d$.\todo[inline]{In the original LDA paper, a document is actually a sequence of N words, where each word is a one-hot vector encoding. Might not be important to change, since we just say the notation is 'similar'.}
```

Comment:
Maybe start by describing W_{d,n} rather than just d and n on their own, to better draw connection to the figure.
```suggestion
Our standard \gls{lda} model uses a similar notation to \citet{blei2003latent}, where $W_{d,n}$ is the $n$'th word in document $d$.
$Z_{d,n}$ is the topic assigned the $n$'th within document $d$.
\todo[inline]{In the original LDA paper, a document is actually a sequence of N words, where each word is a one-hot vector encoding. Might not be important to change, since we just say the notation is 'similar'.}
```
Otherwise
```suggestion
Our standard \gls{lda} model uses a similar notation to \citet{blei2003latent}, where a document $d$ is a vector of $N$ ID's, where the ID represents the given word in the original document.
$Z_{d,n}$ is the topic assigned to a word $n$ within $d$.
\todo[inline]{In the original LDA paper, a document is actually a sequence of N words, where each word is a one-hot vector encoding. Might not be important to change, since we just say the notation is 'similar'.}
```